addons/wowl/static/src/env.ts

```diff
@@ -12,9 +12,19 @@ export async function makeEnv(config: OdooConfig): Promise<OdooEnv> {
     _t,
     systray,
     errorDialogs,
+    debugManagerMenu,
```

Comment:
I would name in debugManager (historical name, no added value to change it imho)